src/arrays/referencing_array_and_its_elements/medium/q11/VowelNames.java

```diff
@@ -0,0 +1,29 @@
+package arrays.referencing_array_and_its_elements.medium.q11;
+
+/*
+Write a Java program that takes a String array of 6 given names. Then, count how many of those names end with the
```

Comment:
Change question text to:
"Write a Java program that asks the user for 6 names and stores them in an array of Strings. Then, count how many of the names end with the letter "B" and display the value.

Sample Output:
    Enter 6 given names:
    Billy Bob Jacob Anthony Bartholomew Angus
    There are 2 names that end with the letter b."